recipe/meta.yaml

```diff
@@ -1,29 +1,42 @@
-{% set version = "1.8.0" %}
-{% set sha256 = "947a314a212b5d94985d89b43440dbe66b696e12bbdf9a2f78967b98d74abedc" %}
+{% set version = "1.14.1" %}
+{% set sha256 = "28d82b0d261544a9bf85b429399929e4986eb00efcf1ce16cc71d269a4c3186c" %}
 
 package:
   name: gstreamer
   version: {{ version }}
 
 source:
-  fn: gstreamer-{{ version }}.tar.xz
   url: https://gstreamer.freedesktop.org/src/gstreamer/gstreamer-{{ version }}.tar.xz
   sha256: {{ sha256 }}
 
 build:
-  skip: true  # [win or osx]
-  number: 2
+  skip: true  # [win]
```

Comment:
Why this shouldn't be skipped for macOS?

Reply:
I copied this over from the anaconda recipes, which does support osx so I would like to leave it. Also, the main reason for the PR, is to add osx support (although it turns out I could pull it from the main anaconda repo, see https://github.com/conda-forge/staged-recipes/pull/5966). But I do admit that I don't have osx. Still, given that it worked in anaconda, it shouldn't be too hard to make it work here, I'm guessing.